### ISSUE TITLE
[Cleanup] Remove references to unused Discovery codebase

### ISSUE THREAD

**Issue by Dee-Pac** [paypal/gimel#79]
These files belong to the Gimel Discovery Service, which is still Work-In-Progress in PayPal & not yet open sourced. In addition, the logic in these files are outdated & hence it does not make sense to have these files in the repo.

https://github.com/paypal/gimel/search?l=Shell
Remove --> gimel-dataapi/gimel-core/src/main/scripts/tools/bin/hbase/hbase_ddl_creator.sh

https://github.com/paypal/gimel/search?l=Python
Remove --> gimel-dataapi/gimel-core/src/main/scripts/tools/bin/hbase/hbase_ddl_tool.py

https://github.com/paypal/gimel/search?l=Text
Remove --> gimel-dataapi/gimel-core/src/main/scripts/tools/bin/hbase/hbase_tables.txt

**Comment by Dee-Pac** [comment]
@Prashant47 👍

**Comment by Rishi55** [comment]
If this is still pending, I would like to complete it.

**Comment by Dee-Pac** [comment]
@Rishi55  Yes, This is still an open issue. We've not see a PR yet on this one. Thanks
